core/src/main/kotlin/links/DRI.kt

```diff
@@ -33,15 +33,17 @@ abstract class DRIExtraProperty<T> {
         ?: (this.javaClass.let { it.`package`.name + "." + it.simpleName.ifEmpty { "anonymous" } })
 }
 
+private val DRI_EXTRA_OBJECT_MAPPER = ObjectMapper()
```

Comment:
What is about `DRIExtraContainer.companion`?

Reply:
Yeah, for some reason I forgot that exists :sweat_smile: Fixed